add_to_app/AndroidAddToAppDemo/app/build.gradle

```diff
@@ -24,6 +24,12 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
             signingConfig signingConfigs.debug
+
+            // Necessary to ensure the libapp.so produced by Shorebird matches the one that ends up
+            // in the APK.
```

Comment:
```suggestion
             // The Dart compiler builds libapp.so directly (does not use clang), and already strips
             // symbols.  However when llvm-strip is run on libapp.so, the symbols are re-sorted
             // causing the hash of the library to change (which can confuse Shorebird tools).
             // This line tells gradle to skip the unnecessary llvm-strip step for libapp.so
             // thus ensuring that the libapp.so that Shorebird sees is byte-identical to the one
             // which ends up in the APK/AAR/AAB.
```